src/brevitas/export/onnx/qonnx/function.py

```diff
@@ -59,6 +59,52 @@ def forward(ctx, x, scale, zero_point, bit_width, narrow_range, signed, rounding
         return y
 
 
+class BrevitasFloatQuantFn(Function):
+
+    @staticmethod
+    def symbolic(
+            g,
+            x,
+            scale,
+            exponent_bit_width,
+            mantissa_bit_width,
+            exponent_bias,
+            has_inf,
+            has_nan,
+            has_subnormal,
+            rounding_mode,
+            max_val):
+        ret = g.op(
+            f'{DOMAIN_STRING}::FloatQuant',
+            x,
+            scale,
+            exponent_bit_width,
+            mantissa_bit_width,
+            exponent_bias,
+            has_inf_i=int(has_inf),
+            has_nan_i=int(has_nan),
+            has_subnormal_i=int(has_subnormal),
+            rounding_mode_s=rounding_mode,
+            max_val_f=max_val)
```

Comment:
`FloatQuant`spec was updated 20241-10-30 to move `max_val` to be a compulsory, broadcastable input and no longer an attribute so this should be adjusted. ref:
https://github.com/fastmachinelearning/qonnx/blob/feature/float_quant/docs/qonnx-custom-ops/floatquant_op.md